src/olympia/migrations/1148-db-schema-cannedresponses.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE `cannedresponses`
+    MODIFY `id` int(10) unsigned NOT NULL,
```

Comment:
`Cannot change column 'id': used in a foreign key constraint 'log_activity_comment_canned_response_id_6a9271d5_fk_cannedres' of table 'olympia_dev_copy.log_activity_comment_draft'`